src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -17,6 +17,8 @@ export default class MultipleProjectSelector extends React.Component {
     projects: PropTypes.array,
     onChange: PropTypes.func,
     onUpdate: PropTypes.func,
+    align: PropTypes.oneOf(['right', 'left']),
+    labelMargin: PropTypes.string,
```

Comment:
Likewise do we really need a separate margin prop for discover/health since we are aligning the designs?